components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -209,12 +216,29 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 
 	_clearSelected() {
 		this._selectedEntries = [];
+		this._hideClearFiltersButton = true;
 	}
 
 	_setSelectedOptions() {
 		this._selectedEntries = [].concat.apply([], Object.values(this._entries || {})).filter(x => x.selected);
 	}
 
+	_refreshFilters() {
+		this._setSelectedOptions();
+		this._hideClearFiltersButton = this._selectedEntries.length < CLEAR_FILTERS_THRESHOLD;
```

Comment:
this property probably not needed any more

Reply:
Good point

Reply:
I got rid of `_hideClearFiltersButton`, in the process I also removed the `_refreshFilters` function